<ui version="4.0" >
 <class>IndirectDiffractionReduction</class>
 <widget class="QWidget" name="IndirectDiffractionReduction" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>212</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>DEMON</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="tabShape" >
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabDemon" >
      <attribute name="title" >
       <string>DEMON</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout" >
       <item>
        <layout class="QGridLayout" name="gridLayout_3" >
         <item row="0" column="0" >
          <widget class="QLabel" name="set_lbInst" >
           <property name="text" >
            <string>Instrument</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="MantidQt::MantidWidgets::InstrumentSelector" name="set_cbInst" >
           <property name="techniques" stdset="0" >
            <stringlist>
             <string>TOF Indirect Geometry Spectroscopy</string>
            </stringlist>
           </property>
          </widget>
         </item>
         <item row="0" column="2" >
          <widget class="QLabel" name="set_lbReflection" >
           <property name="text" >
            <string>Reflection</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3" >
          <widget class="QComboBox" name="set_cbReflection" />
         </item>
        </layout>
       </item>
       <item>
        <widget class="MantidQt::MantidWidgets::MWRunFiles" native="1" name="dem_rawFiles" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>41</verstretch>
          </sizepolicy>
         </property>
         <property name="label" stdset="0" >
          <string>Run Number</string>
         </property>
         <property name="multipleFiles" stdset="0" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_5" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11" >
         <item>
          <widget class="QLabel" name="label" >
           <property name="text" >
            <string>Spectra Min</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="set_leSpecMin" />
         </item>
         <item>
          <widget class="QLabel" name="label_2" >
           <property name="text" >
            <string>Spectra Max</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="set_leSpecMax" />
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_6" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_22" >
         <item>
          <widget class="QCheckBox" name="dem_ckVerbose" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="text" >
            <string>Verbose</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="dem_ckPlot" >
           <property name="text" >
            <string>Plot Result</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="dem_ckSave" >
           <property name="text" >
            <string>Save Result</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10" >
     <item>
      <widget class="QPushButton" name="pbHelp" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text" >
        <string>?</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_14" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbRun" >
       <property name="text" >
        <string>Run</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_11" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbManageDirs" >
       <property name="text" >
        <string>Manage Directories</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MantidQt::MantidWidgets::MWRunFiles</class>
   <extends>QWidget</extends>
   <header>MantidQtMantidWidgets/MWRunFiles.h</header>
  </customwidget>
  <customwidget>
   <class>MantidQt::MantidWidgets::InstrumentSelector</class>
   <extends>QComboBox</extends>
   <header>MantidQtMantidWidgets/InstrumentSelector.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
